source/_integrations/spotify.markdown

```diff
@@ -12,25 +12,25 @@ ha_codeowners:
 ha_domain: spotify
 ---
 
-The `spotify` media player integration allows you to control [Spotify](https://www.spotify.com/) playback from Home Assistant.
+The `Spotify` media player integration allows you to control [Spotify](https://www.spotify.com/) playback from Home Assistant.
 
 ## Prerequisites
 
-- Spotify account
-- Spotify application, properly configured (see below).
+- `Spotify` account
+- `Spotify` application configured for Home Assistant (see [below](#create-a-spotify-application))
 
 <div class='note'>
-  Controlling the Spotify integration (pause, play, next, etc.) requires a Premium account.
-  If you do not have a Premium account, the integration in the frontend will not show the controls.
+  Spotify integrated media controls (play, pause, skip) requires a Premium account.
```

Comment:
It is not limited to media control or play, pause and skip. This change now made it untrue.

I suggest restoring the first part of this sentence.

Reply:
Reverted.